### DISCUSSION TITLE
Home Assistant MQTT - Discovered entities with a name that starts with the device name

### DISCUSSION THREAD

**Discussion by michaeljaddy** [ESPresense/ESPresense#999] [General]
Hi folks,

Have others faced issues with the latest HA updates as per the below and know what the fix would be?

![image](https://github.com/ESPresense/ESPresense/assets/113230724/d32033ef-495c-4740-b1ea-921ead825729)

Many thanks

Mike

**Comment by paul-thomas1964** [comment]
I’ve also noticed this appeared a few days ago.
I assume its advance notice that this will stop working in the 2024.2.0 release, which I’m guessing wont be out until 2024.

I guess there will be code changes needed to the app?

It seems to be discussed further here
[(https://github.com/ESPresense/ESPresense/issues/997)]

**Comment by DTTerastar** [comment]
I am aware of this new wrinkle in Home Assistant.  I will come out with a new firmware to deal w/ the new requirements of Home Assistant.
